solutions/project/project-experimentation/README.md

```diff
@@ -66,7 +66,7 @@ This package has no sub-packages.
 
 1.  Move into the local package:
     ```shell
-    cd "./project-experimentation/"
+    cd ".//solutions/project/project-experimentation/"
```

Comment:
double slash  //solutions to single /
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/main/solutions/project/project-experimentation

Reply:
The generate-kpt-pkg-docs added this in?

Reply:
works either way 
cd ../solutions/project...
cd ..//solutions/project...